src/applications/static-pages/tests/facilities/mockFacilitiesData.js

```diff
@@ -81,4 +81,91 @@ const mockFacilityLocatorApiResponse = {
   ],
 };
 
-module.exports = { mockWidgetFacilitiesList, mockFacilityLocatorApiResponse };
+const mockFeedbackZeroValues = {
+  data: [
+    {
+      id: 'vha_561GE',
+      type: 'facility',
+      attributes: {
+        access: {
+          health: [
+            {
+              service: 'MentalHealthCare',
+              new: 4.333333,
+              established: 22.802325,
+            },
+            {
+              service: 'PrimaryCare',
+              new: 5.545454,
+              established: 0.551282,
+            },
+          ],
+          effectiveDate: '2021-06-21',
+        },
+        activeStatus: 'A',
+        address: {
+          mailing: {},
+          physical: {
+            zip: '07302-3551',
+            city: 'Jersey City',
+            state: 'NJ',
+            address1: '115 Christopher Columbus Drive',
+            address2: null,
+            address3: 'Suite 201',
+          },
+        },
+        classification: 'Primary Care CBOC',
+        detailedServices: null,
+        facilityType: 'va_health_facility',
+        feedback: {
+          health: {
+            primaryCareUrgent: 0.0,
```

Comment:
Maybe change one of these to non-zero just to cover that case?

Reply:
there is already this one https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/static-pages/tests/facilities/FacilityPatientSatisfactionScoresWidget.unit.spec.jsx#L7 , need one more?